### ISSUE TITLE
Bug: Tool `on_drop` function deletes node at player's current position

### ISSUE THREAD

**Issue by Lowie375** [ubc-minetest-classroom/Minetest_Classroom#71]
Affects all tools with the following lines of code:
```lua
on_drop = function(itemstack, dropper, pos)
    minetest.set_node(pos, {name="air"})
end,
```

Instead, to preserve undroppability, `on_drop` should be an function with an empty body - patch is given below
```lua
on_drop = function(itemstack, dropper, pos)
end,
```

**Comment by Lowie375** [comment]
New instance of bug identified on `forestry_tools` branch - will patch

**Comment by Lowie375** [comment]
Should be patched on `forestry_tools` as of commit [`59f83aa`](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/59f83aafea4d86fd293406d2846289819f140eea)
EDIT: re-introduced in commit [`bb94fef`](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/bb94feff35cfe1551a65bbb50f86f5f7e1306361)

**Comment by Lowie375** [comment]
Patch for the instance above merged into develop; closing
